src/current/cockroachcloud/cluster-overview-page.md

```diff
@@ -75,9 +75,9 @@ The **Cluster configuration** panel displays the settings you chose during [clus
 | Cloud     | The cluster's [cloud provider]({% link cockroachcloud/create-your-cluster.md %}#step-2-select-the-cloud-provider). |
 | Plan type | The [plan type]({% link cockroachcloud/create-your-cluster.md %}#step-1-start-the-cluster-creation-process) used to create the cluster.                                                           |
 | Region(s) | The cluster's [region(s)]({% link cockroachcloud/create-your-cluster.md %}#step-3-select-the-region-s).         |
-| Nodes     | The [number of nodes]({% link cockroachcloud/create-your-cluster.md %}#step-4-select-the-number-of-nodes) the cluster has and how many are live. |
-| Compute   | The cluster's [compute power per node]({% link cockroachcloud/create-your-cluster.md %}#step-5-select-the-hardware-per-node).         |
-| Storage   | The cluster's [storage per node]({% link cockroachcloud/create-your-cluster.md %}#step-5-select-the-hardware-per-node).         |
+| Nodes     | The [number of nodes]({% link cockroachcloud/create-your-cluster.md %}#step-5-configure-cluster-capacity) the cluster has and how many are live. |
+| Compute   | The cluster's [compute power per node]({% link cockroachcloud/create-your-cluster.md %}#step-5-configure-cluster-capacity).         |
+| Storage   | The cluster's [storage per node]({% link cockroachcloud/create-your-cluster.md %}#step-5-configure-cluster-capacity).         |
```

Comment:
These are three links to exactly the same section in a row. Is this OK?

Reply:
Fixed one, but the rest are okay.